lib/seafoam/passes/truffle_translators/truffleruby.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Seafoam
+  module Passes
+    module TruffleTranslators
+      class TruffleRuby < Default
+        TRUFFLERUBY_ARGS = [
+          "DECLARATION_FRAME",
+          "CALLER_FRAME_OR_VARIABLES",
```

Comment:
This never contains the the caller frame anymore, but we didn't rename it yet in RubyArguments and simplify it.
It should be:
```suggestion
          "CALLER_SPECIAL_VARIABLES",
```

Reply:
Thanks for the correction.